packages/server/src/lib/email.js

```diff
@@ -47,6 +85,12 @@ async function deliverEmail({
     subject,
     emailType,
 }) {
+    if (!ENABLED_EMAIL_TYPES.includes(emailType)) {
+        log.info({ emailType, enabledEmailTypes: ENABLED_EMAIL_TYPES },
+            'this type of email is not enabled so it will not be sent');
+        return null;
+    }
```

Comment:
This is the actual check that prevents certain/all emails from sending (assuming all emails sent in production do so via this function, which should be the case).